DependencyInjection/Configuration.php

```diff
@@ -26,8 +26,14 @@ public function __construct($debug)
 
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('knp_disqus');
+        $treeBuilder = new TreeBuilder('knp_disqus');
+
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $rootNode = $treeBuilder->getRootNode();
+        } else {
+            // BC layer for symfony/config 4.1 and older
+            $rootNode = $treeBuilder->root('knp_disqus');
+        }
```

Comment:
If we require Symfony 4.4 as the minimum, I think the old way won't be needed anymore.